plugins/modules/source_control/gitlab/gitlab_group_variable.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Florent Madiot (scodeman@scode.io)
+# Based on code:
+# Copyright: (c) 2019, Markus Bergholz (markuman@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_group_variable
+short_description: Creates/updates/deletes GitLab Groups Variables
```

Comment:
```suggestion
short_description: Creates, updates, or deletes GitLab groups variables
```
Titles should be written as a usual sentence (according to the doc). Not sure about the second comma needed in this case.

Reply:
According to https://docs.ansible.com/ansible/latest/dev_guide/style_guide/grammar_punctuation.html#commas I think the comma should be there.

Reply:
thanks!